docs/streams/core-concepts.html

```diff
@@ -160,21 +160,26 @@ <h3><a id="streams_concepts_duality" href="#streams-concepts-duality">Duality of
 
     <p>
         Any stream processing technology must therefore provide <strong>first-class support for streams and tables</strong>.
-        Kafka's Streams API provides such functionality through its core abstractions for
-        <code class="interpreted-text" data-role="ref">streams &lt;streams_concepts_kstream&gt;</code> and
-        <code class="interpreted-text" data-role="ref">tables &lt;streams_concepts_ktable&gt;</code>, which we will talk about in a minute.
-        Now, an interesting observation is that there is actually a <strong>close relationship between streams and tables</strong>,
-        the so-called stream-table duality.
-        And Kafka exploits this duality in many ways: for example, to make your applications
+        Kafka's Streams API provides such functionality through its core abstractions for 
+        <a id="streams_concepts_kstream" href="/{{version}}/documentation/streams/developer-guide/dsl-api#streams_concepts_kstream">streams</a>
+        <code class="interpreted-text" data-role="ref">&lt;streams_concepts_kstreams&gt;</code>
```

Comment:
All the links point to the correct sections but I think we can get rid of all the `<code class="interpreted-text" data-role="ref">&lt;XX&gt;</code>` blocks in this section and elsewhere.  

I could be wrong, but It looked to me the `interpreted-text` sections were placeholders for setting the correct links pointing to other sections in the docs.